app/assets/stylesheets/table/table_panel/add_title_layer.css.scss

```diff
@@ -18,24 +18,76 @@
     @include transition-property("height");
     @include transition-duration(250ms);
 
-    a.info {
-      text-decoration: none;
+    .title_bar {
+      position: relative;
+      width: 100%;
+      height: 45px;
+      top: -10px;
     }
 
-    // Change title
-    // (Title layer text)
-    .action.title {
-      @include inline-block();
-      margin:0 0 0 27px;
-      font:500 17px $text-fonts;
-      line-height:20px;
-      letter-spacing:0;
+    .title_bar {
```

Comment:
Not hurting anything, but no need to close and reopen `.title_bar`.

Also I can't remember if I've seen this specifically mentioned in our house styles, but typical CSS naming conventions call for hyphens rather than underscores.